core/assets-sources/materials/ar_environment_material_flat.mat

```diff
@@ -0,0 +1,35 @@
+material {
```

Comment:
Where is this material used?

Reply:
That is the standard camera material if the DepthMode is disabled